torch/fx/operator_schemas.py

```diff
@@ -44,7 +44,12 @@ def _torchscript_schema_to_signature(ts_schema : torch._C.FunctionSchema) -> ins
     for arg in ts_schema.arguments:
         arg_type = _torchscript_type_to_python_type(arg.type)
         default = arg.default_value if arg.has_default_value() else inspect.Parameter.empty
-        name = arg.name
+        # TODO: Figure out if this is safe. It seems like when generating the type signatures for
```

Comment:
Can you explain a bit more about what the problem is here? I'm not sure what the context is.

Reply:
The schemas use `self` as the name for the first argument, e.g. https://github.com/pytorch/pytorch/blob/407d60ee910175cecc3a95617175105779fbbcbd/aten/src/ATen/native/native_functions.yaml#L424

But `python_torch_functions.cpp` is generated s.t. the first argument is named `input` (e.g. https://github.com/pytorch/pytorch/blob/407d60ee910175cecc3a95617175105779fbbcbd/tools/autograd/templates/python_torch_functions.cpp#L566 and see the generated code in your built repo)

I can't figure out where exactly this switch is happening so the TODO here